### ISSUE TITLE
builtins mutated after importing scapy.arch

### ISSUE THREAD

**Issue by graingert** [secdev/scapy#2680]
#### Brief description

importing scapy.layers.inet (or any module that imports scapy.config or scapy.arch) pollutes built-ins with a lot of things:

#### Environment

-   Scapy version: 2.4.3
-   Python version: py2.7 and py3.7
-   Operating System: Ubuntu 20.04 LTS

#### How to reproduce

 
```python
from six.moves import builtins

old_keys = set(vars(builtins).keys())
import scapy.layers.inet
new_keys = set(vars(builtins).keys())

print(sorted(new_keys - old_keys))
print(six)
```

output:
```
['ETH_P_8021Q', 'ETH_P_ALL', 'IFF_BROADCAST', 'IFF_DEBUG', 'IFF_LOOPBACK', 'IFF_NOARP', 'IFF_NOTRAILERS', 'IFF_POINTOPOINT', 'IFF_PROMISC', 'IFF_RUNNING', 'IFF_UP', 'L2ListenSocket', 'L2Socket', 'L3PacketSocket', 'LINUX', 'LOOPBACK_NAME', 'MTU', 'PACKET_ADD_MEMBERSHIP', 'PACKET_AUXDATA', 'PACKET_BROADCAST', 'PACKET_DROP_MEMBERSHIP', 'PACKET_FASTROUTE', 'PACKET_HOST', 'PACKET_KERNEL', 'PACKET_LOOPBACK', 'PACKET_MR_ALLMULTI', 'PACKET_MR_MULTICAST', 'PACKET_MR_PROMISC', 'PACKET_MULTICAST', 'PACKET_OTHERHOST', 'PACKET_OUTGOING', 'PACKET_RECV_OUTPUT', 'PACKET_RX_RING', 'PACKET_STATISTICS', 'PACKET_USER', 'Packet', 'Padding', 'RTF_REJECT', 'RTF_UP', 'SIOCGIFADDR', 'SIOCGIFCONF', 'SIOCGIFCOUNT', 'SIOCGIFFLAGS', 'SIOCGIFHWADDR', 'SIOCGIFINDEX', 'SIOCGIFNAME', 'SIOCGIFNETMASK', 'SIOCGSTAMP', 'SIOCSIFFLAGS', 'SIOCSIFLINK', 'SOL_PACKET', 'SO_ATTACH_FILTER', 'ScapyInvalidPlatformException', 'Scapy_Exception', 'SuperSocket', 'TP_STATUS_VLAN_VALID', 'VEthPair', 'absolute_import', 'array', 'attach_filter', 'compile_filter', 'conf', 'ctypes', 'get_alias_address', 'get_if', 'get_if_index', 'get_if_list', 'get_if_raw_addr', 'get_if_raw_hwaddr', 'get_iface_mode', 'get_last_packet_timestamp', 'get_working_if', 'in6_getifaddr', 'ioctl', 'os', 'plain_str', 'raw', 're', 'read_routes', 'read_routes6', 'scapy', 'select', 'set_iface_monitor', 'set_promisc', 'six', 'socket', 'struct', 'subprocess', 'time', 'tpacket_auxdata', 'warning']
<module 'scapy.modules.six' from '/usr/share/python/osirium-virtualenv/lib/python2.7/site-packages/scapy/modules/six.pyc'>
```

#### Expected result
```
[]
NameError: name 'six' is not defined
```

**Comment by graingert** [comment]
@gpotter2 it looks like this was introduced here: https://github.com/secdev/scapy/pull/2057/files#diff-c4607426047722c524a9f42908d764ccR439-R467

but I'm not clear why this builtin mutation is needed?

**Comment by gpotter2** [comment]
Good catch. See https://github.com/secdev/scapy/pull/2714

**Comment by gpotter2** [comment]
FYI we'll get rid of it all together in https://github.com/secdev/scapy/pull/2707 but this will only get merged after 2.4.4
